lit_gpt/lora.py

```diff
@@ -543,6 +543,11 @@ def forward(
     def from_name(cls, name: str, **kwargs: Any) -> Self:
         return cls(Config.from_name(name, **kwargs))
 
+    def _init_weights(self, module: nn.Module) -> None:
```

Comment:
A dumb proposal: create `_init_weights` inside `LoRALinear` which calls `reset_parameters` so you can drop this method and two `reset_parameters` calls inside init method for `LoRALinear` and `LoRAQKVLinear`

Reply:
_init_weights is meant to be on the root module, since the scripts do `model.apply(model._init_weights)`

Reply:
Indeed a dumb proposal.
I supposed that `_init_weights` is a PyTorch's special method for each nn.Module class so it can be called recursively (like `reset_parameters`).
Now I get it, now I understand.
Thanks